tests/python/test_files/pylint_test.py

```diff
@@ -1,5 +1,4 @@
 import sys
-import platform  # Unused import
```

Comment:
Restore this line as it was.  We'll find a solution for this line in https://github.com/coala/coala/issues/3066 , after your patch is merged.